lib/AST/GenericSignatureBuilder.cpp

```diff
@@ -4629,8 +4629,33 @@ ConstraintResult GenericSignatureBuilder::addTypeRequirement(
                         ->getDependentType(getGenericParams());
 
       Impl->HadAnyError = true;
+      
+      bool shouldOfferFixit = false;
```

Comment:
Very tiny nitpick: canonically, the camel-case spelling is "FixIt" (capital "I"), from the hyphenated "fix-it".

Reply:
Fixed!